File: app/src/main/java/com/mithilakshar/mithilapanchang/eclipseviewholder.java
package com.mithilakshar.mithilapanchang;

import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

public class eclipseviewholder extends RecyclerView.ViewHolder {

    TextView eclipseName,eclipseDesc;
    ImageView eclipseImage;
    public eclipseviewholder(@NonNull View itemView) {
        super(itemView);

        eclipseName=itemView.findViewById(R.id.eclipseName);
        eclipseDesc=itemView.findViewById(R.id.eclipseDesc);
        eclipseImage=itemView.findViewById(R.id.eclipseImage);
    }
}

File: app/src/main/java/com/mithilakshar/mithilapanchang/eclipseviewholder.kt
package com.mithilakshar.mithilapanchang

import android.view.View
import android.widget.ImageView
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView

class eclipseviewholder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    var eclipseName: TextView
    var eclipseDesc: TextView
    var eclipseImage: ImageView

    init {
        eclipseName = itemView.findViewById(R.id.eclipseName)
        eclipseDesc = itemView.findViewById(R.id.eclipseDesc)
        eclipseImage = itemView.findViewById(R.id.eclipseImage)
    }
}
